libantworld/world.cc

```diff
@@ -34,7 +34,9 @@ void readVector(std::istringstream &stream, std::vector<GLfloat> &vector, float
     }
 
     // Check this is the end of the linestream i.e. there aren't extra components
-    BOB_ASSERT(stream.eof());
+    if(assertIfNotEOF) {
+        BOB_ASSERT(stream.eof());
+    }
```

Comment:
Adding an extra arg just for this assert statement seems like a bit of overkill. How about we just drop the ``BOB_ASSERT`` from here and the caller can check for EOF at the call site if needed?

Reply:
good point